mne/bem.py

```diff
@@ -12,10 +12,512 @@
 from scipy import linalg
 
 from .fixes import partial
-from .utils import (verbose, logger, run_subprocess, get_subjects_dir)
+from .utils import (verbose, logger, run_subprocess, deprecated,
+                    get_subjects_dir)
 from .io.constants import FIFF
+from .io.write import (start_file, start_block, write_float, write_int,
+                       write_float_matrix, write_int_matrix, end_block,
+                       end_file)
+from .io.tag import find_tag
+from .io.tree import dir_tree_find
+from .io.open import fiff_open
 from .externals.six import string_types
-from .surface import read_surface, write_bem_surface
+
+
+# ############################################################################
+# Compute BEM solution
+
+# define VEC_DIFF(from,to,diff) {\
+# (diff)[X] = (to)[X] - (from)[X];\
+
+# The following approach is based on:
+#
+# de Munck JC: "A linear discretization of the volume conductor boundary
+# integral equation using analytically integrated elements",
+# IEEE Trans Biomed Eng. 1992 39(9) : 986 - 990
+#
+
+
+def _calc_beta(rk, rk1):
```

Comment:
That would require (re-)understanding what it does first :) but seriously, I'll add one. It also reminds me that I need to check for duplication of functionality across the `_compute_forward` module, I think these are different but I should check.